docs/reference/faq.md

```diff
@@ -0,0 +1,34 @@
+
+# Frequently Asked Questions
+
+We strive to have most questions answerable by the docs generally, but some questions
+aren't obviously answered by documentation (often because they span multiple topics).
+
+This is the home for such questions! If you have a question not answered here,
+feel free to open an issue or send us a support request.
+
+## Does Broker understand FOSSA CLI config files checked into the repository being scanned?
+
+_This question refers to FOSSA CLI's [`.fossa.yml` config file](https://github.com/fossas/fossa-cli/blob/master/docs/references/files/fossa-yml.md)_
+_and FOSSA CLI's [`fossa-deps` config file](https://github.com/fossas/fossa-cli/blob/master/docs/references/files/fossa-deps.md)._
+
+Broker itself does not, however when Broker runs FOSSA CLI, it does read those config files, so they are respected.
+The main caveat however is that FOSSA CLI does not actually upload the results of the scan to FOSSA; Broker does.
+
+This is done for a few reasons, but primarily in order to offer predictability and observability to the upload process,
+both for FOSSA and for IT organizations, since Broker performs extremely comprehensive tracing of low-level details
+in its debugging information.
```

Comment:
[optional] Minor tweak that gets rid of a "however" and a few commas:

```suggestion
Broker itself does not. However, when Broker runs FOSSA CLI, the CLI does read those config files, so they are respected.
The main caveat is that FOSSA CLI does not actually upload the results of the scan to FOSSA; Broker does.

This is done for a few reasons, but primarily in order to offer predictability and observability to the upload process.
This is useful both for FOSSA and for IT organizations, since this allows Broker to report extremely comprehensive tracing of low-level details
in its debugging information.
```